packages/devtools_server/lib/src/server.dart

```diff
@@ -125,39 +125,57 @@ void serveDevTools({
   //   }
   // }
   _stdinCommandStream.listen((Map<String, dynamic> json) async {
-    final int id = json['id'];
+    final dynamic id = json['id'];
```

Comment:
What types might we get here?

Reply:
VS Code has always used strings. When I changed this, I thought I checked the spec and it said  "any value", however that was actually Wikipedia! The spec says:

> An identifier established by the Client that MUST contain a String, Number, or NULL value if included. 

I don't think we follow JSON-RPC particularly strictly, but I do think we should support both int+string here (I'm using shared code used for Flutter/analysis server/etc. which used strings, and it was just crashing on this line).

I'm hoping [one day](https://github.com/dart-lang/language/issues/145) we can support `int | String`!

Reply:
(I've added a comment about the types)

Reply:
The comment helps a lot. Thanks!